app/redux/ducks/classify.spec.js

```diff
@@ -1,6 +1,12 @@
 import reducer from './classify';
 import { expect } from 'chai';
 import mockPanoptesResource from '../../../test/mock-panoptes-resource';
+import FakeLocalStorage from '../../../test/fake-local-storage';
+
+global.innerWidth = 1000;
+global.innerHeight = 1000;
```

Comment:
Where are these global values being used in the spec?

Reply:
I thought they got added to classification metadata when a classification is created, but maybe not?

Reply:
That makes sense. I was initially confused because I didn't see where `updateClassification` was being used in the Redux store. Looks like the setup in the store is necessary for the tests in `classifier.spec.js` to run correctly.